systemtest/tmt/plans/main.fmf

```diff
@@ -236,22 +236,8 @@ finish:
     test:
       - performance
 
-/performance-capacity:
-  summary: Run performance test suite with kraft
-  provision:
-    hardware:
-      memory: ">= 30 GiB"
-      cpu:
-        processors: ">= 8"
-      # it seems that default disk size (i.e., 50GB) is not enough for capacity tests
-      disk:
-        - size: ">= 60 GB"
-  discover+:
-    test:
-      - performance-capacity
-
-/performance-topic-operator-capacity:
-  summary: Run topic operator performance capacity strimzi test suite
+/capacity:
```

Comment:
do we still need baremetal machines for performance plan? And for capacity?

Reply:
Good point. I think we don't need it anymore for performance. But for capacity, I think yes.